internal/converter/internal/otelcolconvert/testdata/probabilistic_sampler.alloy

```diff
@@ -11,6 +11,8 @@ otelcol.receiver.otlp "default" {
 }
 
 otelcol.processor.probabilistic_sampler "default" {
+	fail_closed = true
+
```

Comment:
you need to remove these two lines now that the default value is the same between otel and alloy